.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules/
+/test.md/
```

Comment:
- [ ] Your test.md file can still be seen on your repository because you did not ignore it properly inside your .gitignore file. Kindly use [this](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) file to untrack them first. Then check [this](https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/) article to learn how to ignore both the folders and files in the `.gitignore` file